modules/core/shared/src/main/scala/gem/syntax/ShapeExpression.scala

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.syntax
+
+import gem.geom.ShapeExpression
+import gem.geom.ShapeExpression._
+
+import gem.math.{Angle, Offset}
+
+final class ShapeExpressionOps(val self: ShapeExpression) extends AnyVal {
+
+  def difference(that: ShapeExpression): ShapeExpression =
+    Difference(self, that)
+
+  def intersection(that: ShapeExpression): ShapeExpression =
+    Intersection(self, that)
+
+  def rotate(a: Angle): ShapeExpression =
+    Rotate(self, a)
+
+  def translate(o: Offset): ShapeExpression =
+    Translate(self, o)
+
+  def union(that: ShapeExpression): ShapeExpression =
+    Union(self, that)
+
+  def -(that: ShapeExpression): ShapeExpression =
+    difference(that)
```

Comment:
Isn't the set difference operator typically `∖`?

Reply:
I guess it doesn't really matter. People will probably better know what `-` means.